.devcontainer/dev/devcontainer.json

```diff
@@ -1,10 +1,44 @@
 {
-  "name": "uyuni-dev-container",
+  "name": "uyuni-devcontainer",
```

Comment:
Aha ok, well, it might be mixed with the devcontainer we will have in .devcontainer/test folder. But ok, I doubt we run both at same time.